src/ui/NoNameFacet/NoNameFacetHeader/NoNameFacetHeaderButton.ts

```diff
@@ -0,0 +1,44 @@
+import { $$ } from '../../../utils/Dom';
+import { SVGDom } from '../../../utils/SVGDom';
+
+export interface INoNameFacetHeaderButtonOptions {
+  label: string;
+  action?: () => void;
+  shouldDisplay: boolean;
+  className: string;
+  iconSVG: string;
```

Comment:
I would remove the option to add an icon in the facet header.
It's there in the old facet for legacy reason only.

Reply:
not sure I understand here

Reply:
NVM, misunderstood.